modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -295,6 +295,107 @@ a|
 label:functionality[]
 label:new[]
 
+[source, cypher, role="noheader"]
+----
+CREATE [COMPOSITE] DATABASE actors DEFAULT LANGUAGE CYPHER 25
+----
+
+a| Set the default Cypher version for a standard or composite database when creating it.
+The available versions are `CYPHER 25` and `CYPHER 5`.
+If not specified, the default language for the database is set to the default language of the DBMS.
+For more information, see the following sections in the Operations Manual:
+
+* link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/#_create_a_database_with_default_language[Create a database with a `DEFAULT LANGUAGE`].
+* link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/composite-databases/create-composite-databases/#composite-databases-default-language[Set the default Cypher version for a composite database].
+
+a|
+label:functionality[]
+label:new[]
+
+[source, cypher, role="noheader"]
+----
+CREATE ALIAS `remote-with-default-language` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+DEFAULT LANGUAGE CYPHER 25
```

Comment:
```suggestion
CREATE ALIAS `remote-with-default-language`
FOR DATABASE `northwind-graph-2020`
  AT "neo4j+s://location:7687"
  USER alice
  PASSWORD 'example_secret'
  DEFAULT LANGUAGE CYPHER 25
```
This might be a nicer formatting 🤔 makes it more visible that the AT, USER, PASSWORD and DEFAULT LANGUAGE clauses are part of the FOR DATABASE 🤔